README.md

```diff
@@ -381,6 +382,9 @@ bin/zxp config _config-file_
 ## Enable Metrics
 In the zot with all extensions case see [configuration example](./examples/config-metrics.json) for enabling metrics
 
+## Enable Mandatory Annotations
```

Comment:
Call this "Image linting" and add a subsection which says annotations, etc. The idea is we can add more subsections later under the linting umbrella.